pmd-packager/upgrade.sh

```diff
@@ -1,5 +1,7 @@
-VERSION=$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | jq --raw-output ".tag_name" | sed 's:.*/::')
+VERSION=$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | grep '.tag_name' | sed 's:.*/::' | sed 's:",::')
 echo $VERSION
+sed -i "" "s|version\>[0-9\.]*\<|version>$VERSION<|g" pom.xml
```

Comment:
We will need to update pom.xml because it is used in this packaging process using Maven